doc/nrf/glossary.rst

```diff
@@ -0,0 +1,426 @@
+   These are foundational options for the application that will always be applied to its built image.
+   However, they can be overridden by applying an additional Kconfig fragment at build time.
+
+Link Layer (LL)
+   “A control protocol for the link and physical layers that is carried over logical links in addition to user data.” Bluetooth Core Specification, Version 5.1, Vol 1, Part A, Section 1.2
+
+Low-Noise Amplifier (LNA)
+   In a radio receiving system, an electronic amplifier that amplifies a very low-power signal without significantly degrading its signal-to-noise ratio.
+
+LTE-M
+   An open standard that is most suitable for medium-throughput applications requiring low power, low latency, and/or mobility, like asset tracking, wearables, medical, POS, and home security applications.
+   Also known as Cat-M1.
+
+Lightweight Machine to Machine (LwM2M)
+   An application layer protocol.
+   It defines the service architecture for IoT devices and the protocol for device management.
+
+Man-in-the-Middle (MITM)
+   A man-in-the-middle attack is a form of eavesdropping where communication between two devices is monitored and modified by an unauthorized party who relays information between the two devices giving the illusion that they are directly connected.
+
+Manifest file
+   The file containing the metadata of the manifest repository.
+   In NCS and Zephyr, it is :file:`west.yml`.
+
+Manifest repository
+   A repository that contains a :file:`west.yml` file in its root folder and can therefore act as the center of a repository star topology.
+
+Mass Storage Device (MSD)
+   Any storage device that makes it possible to store and port large amounts of data in a permanent and machine-readable fashion.
+
+Maximum Transmission Unit (MTU)
+   The largest packet or frame that can be sent in a single network-layer transaction.
+
+MCUboot
+   A secure bootloader for 32-bit microcontroller units, which is independent of hardware and operating system.
+
+Message Queue Telemetry Transport (MQTT)
+   A machine-to-machine (M2M) connectivity protocol used by some IoT devices.
+   It is designed as an extremely lightweight publish/subscribe messaging transport.
+   It is useful for connections with remote locations where a small code footprint is required and/or network bandwidth is at a premium.
+   For example, it has been used in sensors communicating to a broker via satellite link, over occasional dial-up connections with healthcare providers, and in a range of home automation and small device scenarios.
+
+Microcontroller Unit (MCU)
+   A small computer on an integrated circuit.
+
+Menuconfig
+   A tool to configure kernel source.
+   It was initially created for the Linux kernel.
+   It uses the Kconfig configuration language.
+
+Multicast addressing
+   An addressing type that uses a one-to-unique many association, where datagrams are routed from a single sender to multiple selected endpoints simultaneously in a single transmission.
```

Comment:
>one-to-unique many

Can many actually be unique? I think this part can safely be replaced with `one-to-many`

Reply:
fixed